efel/cppcore/LibV2.cpp

```diff
@@ -294,30 +326,38 @@ int LibV2::AP_rise_rate(mapStr2intVec& IntFeatureData,
 
 // *** AP_fall_rate according to E12 and E20 ***
 static int __AP_fall_rate(const vector<double>& t, const vector<double>& v,
+                          const double stimstart,
                           const vector<int>& peakindices,
                           const vector<int>& apendindices,
                           vector<double>& apfallrate) {
   apfallrate.resize(std::min(apendindices.size(), peakindices.size()));
+  vector<int> newpeakindices;
+  for (size_t i=0; i < peakindices.size(); i++) {
+    if (t[peakindices[i]] > stimstart) {
+      newpeakindices.push_back(peakindices[i]);
+    }
+  }
```

Comment:
This pattern occurs in multiple places. It can be reused now and also in the future. Can you put it into a function?

Reply:
Implemented in latest commit. Tell me if it looks good to you